### PR TITLE
Ml module strategy plat 423

### PR DESCRIPTION
**Pull request by bobturneruk**
# Feature

**_WHAT DOES THIS PR DO?_**

Adds information on how code and data are shared between the online platform and offline R&D activities around Machine Learning.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**

PLAT-423

**_WHAT IS THE LINK TO THE PR BRANCH_**

https://github.com/airqo-platform/AirQo-modules/tree/ml-module-strategy-PLAT-423

**_HOW DO I TEST OUT THIS PR?_**

Documentation only - does the documentation accurately reflect our practise?

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_**

NA

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

NA

**_ARE THERE ANY RELATED PRs?_**

No

### CODE REVIEW

**Comment by bobturneruk** [comment]
@DennisReddyhoff can you pick up this branch, please, and document your proposals for `data/`, `prediction_models/` and `trainers/`, please? Feedback on what I've done so far is also welcome. I'm trying (as usual) to document things as they are as a prelude to future planning.

**Comment by bobturneruk** [comment]
(Or if there's existing docs / meeting notes then remind me where!)

File: README.md [hunk 1]
```diff
@@ -18,6 +24,33 @@ To install the dev build, clone the repo and use
...
+
+This module is under development and is planned to provide an absraction layer (unifying calls to BigQuery, mongodb and AirQo API) along with utility functions to be shared by multiple machine learning models e.g. GPS co-ordinate transforms.
+
+### Adding Machine Learning Models
+
+The process for adding machine learning models to the AirQo platform is summarised here.
+
+![add-ml-process](assets/add-ml-process.svg)
+
+Data is manually extracted from the AirQo platform (by a combination of direct access to BigQuery and mongodb, and by API calls). The data is then manually "wrangled" into a form that can be used in development of Machine Learning models. I is typically split into test and training datasets. The training dataset is used to train the model, yielding (hyper)parameters that define the fit of the model to the data. It is validated using the test dataset which provides metrics of model performance.
```
Comment (Baalmart) [inline README.md:48]: Awesome explanation, great work.

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @bobturneruk

File: README.md (hunk 1 above)
Comment (bobturneruk) [reply README.md:48]: Thanks @Baalmart - I think @DennisReddyhoff is going to add to this soon.

**Comment by DennisReddyhoff** [comment]
I've added to the README to better explain the submodules. The data/ folder was intended to provide a default file structure for the data_models but I'm not actually sure it's required. I can add a quick note about it maybe?

**Comment by bobturneruk** [comment]
I think this is at the stage where it can be merged on approval, although improvements and additions could be made further down the line.

**Review by sserurich — APPROVED** [review]
This looks ok for now.

**Review by Priscilla-AL — APPROVED** [review]

**Comment by bobturneruk** [comment]
Thanks all. Now there are 3 approvals I will merge.

**Pull request merged**